tests/acceptance/conftest.py

```diff
@@ -56,11 +56,16 @@ def pytest_configure(config):
     """
     )
 
-    subprocess.call(
-        ["yarn", "webpack"],
-        env={
-            "NODE_ENV": "development",
-            "PATH": os.environ["PATH"],
-            "NODE_OPTIONS": "--max-old-space-size=4096",
-        },
-    )
+    try:
+        subprocess.call(
+            ["yarn", "--silent", "webpack"],
+            env={
+                "NODE_ENV": "development",
+                "PATH": os.environ["PATH"],
+                "NODE_OPTIONS": "--max-old-space-size=4096",
+            },
+        )
+    except OSError:
+        print(
+            "Unable to find `yarn` and/or `webpack` -- make sure your development environment is setup correctly: https://docs.sentry.io/development/contribute/environment/#macos---nodejs"
```

Comment:
Not necessarily limited to not found - what kind of error prompted you to add this?

FWIW direnv does this:

```
if [ ! -x "node_modules/.bin/webpack" ]; then
    info "You don't seem to have yarn packages installed."
    advice_install_yarn_pkgs
fi
```

And advice_install_yarn_pkgs will tell you to run `make install-yarn-pkgs` which uses vendored yarn.

Reply:
![image](https://user-images.githubusercontent.com/79684/79377822-26c77980-7f11-11ea-9cb3-312b58b401bf.png)

This is what happens when yarn is not found

Reply:
I've updated to check status code when running yarn webpack, since it won't throw a python error, but it'll return with a non zero status code.

Also instead of print, raise an exception in both cases so tests do not continue to run.

Reply:
Huh, that particular OSError not what I would have expected. Do you know what file/dir it's trying to find?

Reply:
Ya instead of `yarn` I changed it to `yarn2` lol

Reply:
lol